magefiles/mage.go

```diff
@@ -143,6 +143,31 @@ func Clean() error {
 			return err
 		}
 	}
+
+	// cleanup rust for public builds
+	env, err := getEnv()
+	if err != nil {
+		return err
+	}
+	if !strings.Contains(env["FEATURES"], "internal") {
```

Comment:
I dont think it is necessary (or even totally correct) to check for `internal` - IMO, it's sufficient to check for dir presence - if it is present - clean it. Just add these dirs to the `buildFolders` list on line #106

Reply:
Addressed here https://github.com/NordSecurity/nordvpn-linux/pull/575/files